scripts/ci_install_dependency.sh

```diff
@@ -3,9 +3,8 @@ Install the dependency in CI.
 """
 
 pip install --upgrade pip
-pip install -e "python[all]"
+pip install -e "python[all]" --find-links https://flashinfer.ai/whl/cu121/torch2.4/flashinfer/
```

Comment:
Why not use `--extra-index-url `? Because https://github.com/flashinfer-ai/whl is HTML
When I use `pip index versions flashinfer --index-url https://flashinfer.ai/whl/cu121/`
```
WARNING: The directory '/root/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you should use sudo's -H flag.
WARNING: pip index is currently an experimental command. It may be removed/changed in a future release without prior warning.
flashinfer (0.0.1+cu121)
Available versions: 0.0.1+cu121
  INSTALLED: 0.1.6+cu121torch2.4
  LATEST:    0.0.1+cu121
```
It can't get the right `LATEST`